### PR TITLE
User can enter job title and role start date in profile.

### PR DESCRIPTION
**Pull request by robmckinnon**

### CODE REVIEW

**Comment by novotnyjakub** [comment]
It seems that you've removed support for the `organisation` field, but it's still in allowed params and as a column in the database.

**Comment by robmckinnon** [comment]
I'll remove `organisation` from allowed params.

File: db/schema.rb [hunk 1]
```diff
@@ -62,11 +62,12 @@
     t.text     "email"
     t.text     "job_title"
     t.integer  "manager_id"
-    t.datetime "created_at",             null: false
-    t.datetime "updated_at",             null: false
+    t.datetime "created_at",              null: false
+    t.datetime "updated_at",              null: false
     t.string   "staff_number"
     t.string   "grade"
     t.string   "organisation"
```
Comment (novotnyjakub) [inline db/schema.rb:69]: The field is still here, do you want to preserve it in case it's used later?

Comment (robmckinnon) [inline db/schema.rb:69]: Think it's ok to preserve it for now. HR SCS requirements usage may come back?

**Pull request merged**